dangerfile.js

```diff
@@ -25,12 +25,26 @@ import { danger, warn, message } from 'danger';
 
 const fileChanges = [...danger.git.modified_files, ...danger.git.created_files];
 
+// Always be nice.
+message('Thanks for the PR 🎉.');
+
 // If any of the packages have changed, the changelog should have been updated.
 const changelogModified = includes(fileChanges, 'changelog.md');
-const packagesModified = fileChanges.filter(filePath => filePath.startsWith('packages/')).length > 0;
+const packagesModified = fileChanges.filter(filePath => (
```

Comment:
Could use `Array.prototype.some` here instead

Reply:
I didn't know about `some`! That's awesome